InOffice/View/ProfileDetailsViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ProfileDetailsViewController.swift
+//  InOffice
+//
+//  Created by Luka Muše on 15.12.2021..
+//
+
+import UIKit
+
+class ProfileDetailsViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        self.title = "Profile"
+        setupViews()
+        setupConstraints()
+    }
+
+    let users: UserModel
+
+    init(users: UserModel) {
+        self.users = users
+        super.init(nibName: nil, bundle: nil)
```

Comment:
since you are creating views programmatically, don't use storyboard init constuctor